packages/sdk/observability/src/providers/ip-data.ts

```
@@ -0,0 +1,87 @@
+        city: ipData.city,
+        region: ipData.region,
+        country: ipData.country,
+        latitude: ipData.latitude,
+        longitude: ipData.longitude,
+      });
+    }).pipe(
+      Effect.provide(FetchHttpClient.layer),
+      Effect.catchAll((err) =>
+        Effect.gen(function* () {
+          log.catch(err);
+        }),
```

Biome (2.3.13)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)